src/actions/index.js

```diff
@@ -1,26 +1,29 @@
-import axios from 'axios';
+import {db} from '../config';
 
 export const FETCH_BEER = 'FETCH_BEER';
 export const UPDATE_BEER = 'UPDATE_BEER';
 
 const url = 'http://localhost:3000/beers'
 
-export function fetchBeers(){
+export function fetchBeers() {
+    var data = db.collection('Beers').get().then((snapshot) => {
+        return snapshot.docs.map((doc) => {
+            return { "id": doc.id, "data": doc.data() };
```

Comment:
you can remove `return` keywords, they are not necessary

Reply:
without return the promise value isn't passed back to data

Reply:
`then((snapshot) => snapshot.docs.map((doc) => ({"id": doc.id, "data": doc.data()}))`